site/content/contribute/getting-started/core-committers.md

```diff
@@ -26,7 +26,7 @@ Below is the list of core committers working on Mattermost:
     - Dev areas: Mobile / Desktop and Web apps
 - **<a name="elias.nahum">Elias Nahum</a>**
     - @elias on [community.mattermost.com](https://community.mattermost.com/core/messages/@elias) and [@enahum](https://github.com/enahum) on GitHub
-    - Dev areas: Mobile / Desktop and Web apps
+    - Dev areas: Mobile apps
```

Comment:
```suggestion
    - Dev areas: Mobile app
```

Reply:
Well there is more than one, I'll change it to just mobile